hms-api/__tests__/service/idea-service.test.ts

```diff
@@ -515,10 +485,9 @@ describe('Get Idea Response', () => {
       ReferenceNotFoundError,
     );
     expect(mockGetIdea).toHaveBeenCalledWith(idea.id);
-    expect(mockGetParticipant).toHaveBeenCalledWith(idea.ownerId);
-    expect(mockGetUser).toHaveBeenCalledWith(ownerParticipant.userId);
+    expect(mockGetUser).toHaveBeenCalledWith(idea.ownerId);
     expect(mockGetParticipants).toHaveBeenCalledWith(idea.participantIds);
-    expect(mockGetUsers).toHaveBeenCalledWith([ownerUser.id, user.id]);
+    expect(mockGetUsers).toHaveBeenCalledWith([undefined, user.id]);
```

Comment:
this seems wrong 🤔

Reply:
I know but I absolutly cant find the problem there

Reply:
checked again and I really dont get it

Reply:
ok, I can have a look

Reply:
thx a lot